### PR TITLE
Swift UI now includes creating and joining channels.

### PR DESCRIPTION
**Pull request by meshula**
Also includes many fixes from debugging session. Messages not yet displayed.

### CODE REVIEW

File: swift/Multiplay/Multiplay.swift [hunk 1]
```diff
@@ -3,39 +3,82 @@ import Foundation
...
+        print("Request timed out for: ", url.absoluteString)
+    }
+    return json
+}
+
+func authenticate(url:URL?) -> JSON? {
+    guard let url = url else { return nil }
+    let request = URLRequest(url:url)
+    let config = URLSessionConfiguration.default
+    let userPassword = "StarLord89:69"
```
Comment (jacques-gasselin) [inline swift/Multiplay/Multiplay.swift:33]: Nice, perhaps a comment here on HTTP-Basic

Comment (jacques-gasselin) [reply swift/Multiplay/Multiplay.swift:33]: I imagine a tiny SwiftUI password dialog here would work.

**Review by jacques-gasselin — APPROVED** [review]

**Comment by jacques-gasselin** [comment]
Is it ready to merge or have you got a few more bits you want to add?

File: swift/Multiplay/Multiplay.swift (hunk 1 above)
Comment (meshula) [reply swift/Multiplay/Multiplay.swift:33]: Oh dang, I left the testing code in. Yes, a dialog is required.

**Comment by meshula** [comment]
Before merging I'd like to touch base on authentication; as you've pointed out the test code -- it highlights that we should sort out the "right thing"

**Review by jacques-gasselin — APPROVED** [review]

**Pull request merged**